### ISSUE TITLE
Is 3ds example PCI compliant? 

### ISSUE THREAD

**Issue by roeeyud** [chargebee/chargebee-checkout-samples#67]
https://github.com/chargebee/chargebee-checkout-samples/blob/master/3ds-examples/chargebee-test/card/index.html
In the file linked about you can see that the page doesn't use ChargeBee components, isn't that not PCI compliant?

**Comment by cb-dinesh** [comment]
Yes, if you are opting for direct 3DS integration, you are expected to be PCI Compliant. For non PCI compliant use-cases, you can opt for [Chargebee Components](https://github.com/chargebee/chargebee-checkout-samples/tree/master/components#chargebee-components)

**Comment by roeeyud** [comment]
OK so I can't find any example using a ChargeBee component and authorizeWith3ds and that API doesn't seem to work

**Comment by cb-samuel** [comment]
@roeeyud Please find attached a sample snippet of the flow which you had requested. Feel free to write to support@chargebee.com for more assistance reg this.
[example.txt](https://github.com/chargebee/chargebee-checkout-samples/files/6350132/example.txt)

**Comment by roeeyud** [comment]
> @roeeyud Please find attached a sample snippet of the flow which you had requested. Feel free to write to [support@chargebee.com](mailto:support@chargebee.com) for more assistance reg this.
> [example.txt](https://github.com/chargebee/chargebee-checkout-samples/files/6350156/example.txt)

Thanks for the reply @cb-samuel , I have very similar code and I keep getting back and error "Invalid card number" on my test site when using cards from https://www.chargebee.com/docs/chargebee-test-gateway.html

What might I be missing?

**Comment by cb-samuel** [comment]
@roeeyud Could you please write to Support@chargebee.com, so our team can analyze and suggest a solution for this.
